user-dao-intro.adoc

```diff
@@ -0,0 +1,144 @@
+
+* Contributors _maintain_ Bisq through valuable work
+* Traders _use_ Bisq to buy and sell bitcoin and other cryptocurrencies
+* Contributors _earn_ BSQ tokens when their compensation requests are approved through voting
+* Traders _buy_ BSQ tokens in order to pay lower trading fees
+* Stakeholders _vote_ on compensation requests
+* Contributors can _lock_ a bond in BSQ to ensure integrity in high-trust roles
+
+Now let's see how these dynamics enable the Bisq DAO to provide 3 core governance functions without any central points of authority.
+
+=== Earn and distribute revenue
+
+Every project needs to be financial sustainable to fund ongoing operations and development. But traditional means of handling revenue and revenue distribution are necessarily centralized—any receiving address or account must be owned by one person, or a small group; determining how much a person should be paid must be determined by one person, or a small group; etc.
+
+The Bisq DAO seeks to avoid such centralization of authority. So, instead of taking in trading fees and then distributing them, the Bisq DAO distributes fees first and then puts them up for sale.
+
+Here's how it works: after a Bisq contributor does work, they file a **compensation request** in the DAO with a description of what they did and how much BSQ they want in return. Then, stakeholders (who are other contributors, traders, and anyone else with BSQ) vote for/against the request. If the request is approved, the contributor is issued new BSQ in the amount they requested.
+
+[NOTE]
+.Where does new BSQ actually come from?
+====
+Recall that a BSQ token is merely colored bitcoin. When a contributor makes their compensation request in the DAO, they must also include the tiny amount of bitcoin to be 'colored' as BSQ (<<dao/specification#,the spec>> requires 100 satoshis). So if a contributor requests 1,000 BSQ, they will need to include 100 * 1,000 = 100,000 satoshis with their compensation request—just about 3 USD at current rates.
+
+If their request is approved, those satoshis are 'colored' and recognized as 1,000 valid BSQ tokens on Bisq. Assuming a BSQ market value of 1 USD (exact value will fluctuate), the contributor will have been granted 1,000 USD worth of BSQ for a negligible cost of 3 USD.
+====
+
+Then, a trader looking for lower trading fees can buy those BSQ tokens from the contributor. When they buy BSQ tokens for BTC, the contributor is paid for their work, and the value transfer is complete!
+
+In this way, there is no need for a central entity to collect and distribute revenue: the BSQ token enables a transfer of value from producer to consumer without any single entity controlling any aspect of the decision-making or distribution process.
+
+Please note: the Bisq DAO does not _require_ traders to use BSQ for trading fees. They're free to pay trading fees directly with BTC, but they will pay higher rates than if they bought BSQ with BTC and paid with BSQ instead.
+
```

Comment:
Maybe we need to add info where those BTC fees end up? 
Short answer: the arbitrators
Long answer: with new trade protocol (comes some time after dao launch) they are sent to a "donation" address determined by voting (e.g. Bisq donation address). From there BTC can be used to buy BSQ on the market and then burn it and by that distributing it to all stakeholder.